docs/docs/guides/labs/components/components-etl-pipeline-tutorial.md

```diff
@@ -49,14 +49,30 @@ See the [`duckdb`](https://duckdb.org/docs/installation/?version=stable&environm
 
 ### 2. Scaffold a new project
 
-After installing dependencies, scaffold a components-ready project:
+After installing dependencies, scaffold a components-ready project. The flow for scaffolding a project will depend on your package manager/environment management strategy.
 
-<CliInvocationExample path="docs_snippets/docs_snippets/guides/components/index/2-scaffold.txt" />
-
-The `dg scaffold project` command builds a project at `jaffle-platform` and initializes a new Python
-virtual environment inside it. When you use `dg`'s default environment management behavior, you won't need to worry about activating this virtual environment yourself.
+<Tabs>
+    <TabItem value="uv" label="`uv` (global `dg`)">
+        Ensure you have `dg` installed globally as a `uv` tool. Then run: 
+        <CliInvocationExample path="docs_snippets/docs_snippets/guides/components/index/2-a-uv-scaffold.txt" />
+        The `dg init` command builds a project at `jaffle-platform`. We passed the `--uv-sync` option, which tells `dg` to immediately run `uv sync` after scaffolding to create a virtual environment and install the dependencies listed in `pyproject.toml`. Now lets enter the directory and activate the virtual environment:
+        <CliInvocationExample path="docs_snippets/docs_snippets/guides/components/index/2-b-uv-scaffold.txt" />
+    </TabItem>
+    <TabItem value="pip" label="`pip` (project-scoped `dg`)">
+        Because `pip` does not support global installations, you will install `dg` inside your project virtual environment.
+        We'll create and enter our project directory, initialize and activate a virtual environment, and install the `dagster-dg` package into it:
+        <CliInvocationExample path="docs_snippets/docs_snippets/guides/components/index/2-a-pip-scaffold.txt" />
+        <CliInvocationExample path="docs_snippets/docs_snippets/guides/components/index/2-b-pip-scaffold.txt" />
+        <CliInvocationExample path="docs_snippets/docs_snippets/guides/components/index/2-c-pip-scaffold.txt" />
+        <CliInvocationExample path="docs_snippets/docs_snippets/guides/components/index/2-d-pip-scaffold.txt" />
+        The `dg` executable is now available via the activated virtual environment. Let's run `dg init .` to scaffold a new project. The `.` tells `dg` to scaffold the project in the current directory.
+        <CliInvocationExample path="docs_snippets/docs_snippets/guides/components/index/2-e-pip-scaffold.txt" />
```

Comment:
This snippet `/.../jaffle-platform`. The ellipsis is confusing

Reply:
This is kinda tricky because:

- the real-world output will show the absolute path (which is good)
- we mask this throughout this tutorial (and our other docs whenever we do something similar) because the snippets are generated in a temporary directory

I agree it looks weird in this case but would like to punt on changing it in this PR

Reply:
`/path/to/your/projects/jaffle-platform`

Reply:
wdyt of just `.../jaffle-platform`? In any case we should do that in a follow-up because it will affect a large number of other docs since they all use the same shortening code

Reply:
sure